codespell_lib/data/dictionary.txt

```diff
@@ -26746,6 +26799,8 @@ provdied->provided
 provdies->provides
 provding->providing
 provedd->proved, provided,
+provence->province
```

Comment:
What about the case of the [Provence](https://en.wikipedia.org/wiki/Provence) province in France? Perhaps a corner case?